src/ior.h

```diff
@@ -215,6 +217,7 @@ typedef struct
 typedef struct {
    double *writeTime;
    double *readTime;
+   size_t pairs_accessed; // number of I/Os done, useful for deadlineForStonewalling
```

Comment:
Whoever implemented stonewalling made a very unfortunate choice in referring to transfers as "pairs."  I would like to call these "transfers" to be consistent with the `transferSize` option; what do you think?

Reply:
Totally agree @glennklockwood .  Pairs is very unintuitive.